### PR TITLE
Update robolectric contributing

### PR DESCRIPTION
**Pull request by utzcoz**

### CODE REVIEW

**Comment by utzcoz** [comment]
I found the contributing of website is obsolete, so I updated it with this PR. @hoisie could you help to review it ? Thanks.

File: contributing.md [hunk 1]
```diff
@@ -25,16 +25,12 @@ Set Android environment variables:
...
-    
-Copy all required Android dependencies to your local Maven repository:
-
-    ./scripts/install-dependencies.rb
 
-Perform a full build of all shadows:
+Robolectric is built using Gradle. Both IntelliJ and Android Studio can import the top-level build.gradle file and will automatically generate their project files from it.
 
-    ./gradlew clean assemble install compileTest
+Robolectric supports running tests against multiple Android API levels. The work it must do to support each API level is slightly different, so its shadows are built separately for each. To build shadows for every API version, run:
```
Comment (hoisie) [inline contributing.md:31]: Can you remove the sentence `The work it must do to support each API level is slightly different, so its shadows are built separately for each.`. I think that applies to a much older version of Robolectric that actually used java templates to have separate shadow files per api level.

File: contributing.md [hunk 2]
```diff
@@ -49,6 +45,8 @@ Essentially the IntelliJ default Java style, but with two-space indents and Goog
 5. No `'m'` or `'s'` prefixes before instance or static variables.
 6. Import Google's [java imports style](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) ([IntelliJ style file here](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml)).
 
+If your changes break the code style, the CI will fail, and your CL will be blocked. You can use [google-java-format](https://github.com/google/google-java-format) to format your code locally before you push your changes for reviewing.
```
Comment (hoisie) [inline contributing.md:48]: May be useful to have a link to https://github.com/robolectric/robolectric/wiki/Running-google-java-format.

**Comment by utzcoz** [comment]
Hi @hoisie , it is ready for review, could you help to review it ? Thanks.

**Comment by utzcoz** [comment]
Hi @hoisie , could you help to review this PR? Thanks.

**Pull request merged**